queries.sql

```diff
@@ -16,3 +16,32 @@ UPDATE animals SET species = 'unspecified';
 SELECT * FROM animals; -- Verify that the change was made
 ROLLBACK;
 SELECT * FROM animals; -- Verify that the species column went back to the state before the transaction
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals; -- Verify that the changes were made
+COMMIT;
+SELECT * FROM animals; -- Verify that the changes persist after commit
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals; -- Verify that the changes were made
+ROLLBACK;
+SELECT * FROM animals; -- Verify that the changes were reverted
+
+BEGIN;
+DELETE  FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals; -- Verify that the changes were made
+
+SELECT COUNT(*) FROM animals;
```

Comment:
- [ ] Please kindly include the screenshot of this query to the PR as well as stated in the requirement. 🙏 

<img width="448" alt="Untitled" src="https://github.com/lucy-sees/VeterinaryVentures/assets/103386183/8d463ed8-c52d-458c-8f16-8724820677d8">